js/tabs.js

```diff
@@ -465,12 +469,28 @@ Tabs.prototype.saveEntry_ = function(tab, entry, opt_callback) {
   this.save(tab, opt_callback);
 };
 
-Tabs.prototype.onDocChanged_ = function(e, session) {
+/**
+ * @param {Event} event The DocChange event object.
+ * @param {Object} eventData The data passed with the docChange event.
+ * The event handler which is called when a docChange event is caught, this
+ * handler registers the change and then syncs the codemirror and textarea
+ * instances so they both contain the same text. The event data object contains
+ * where the event came from, i.e textarea or codemirror, which helps the
```

Comment:
maybe move this info about the eventData object to the param definition for it.

Reply:
will do